front/components/assistant/conversation/actions/AgentActionsPanel.tsx

```diff
@@ -0,0 +1,97 @@
+import { Spinner } from "@dust-tt/sparkle";
```

Comment:
really just a move of `front/components/assistant/conversation/actions/AgentMessageActionsDrawer.tsx`
 here, adapted to new archi, so smaller than it seems